### PR TITLE
test: add unit test for global min gas price getter

### PR DESCRIPTION
**Pull request by ninabarbakadze**
## Overview

Added a unit test for GlobalMinGasPrice constant getter. https://github.com/celestiaorg/celestia-app/pull/3040#discussion_r1465446053

Refactored existing unit tests to make it more readable.

### CODE REVIEW

**Review by rootulp — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]

File: pkg/appconsts/versioned_consts_test.go [hunk 1]
```diff
@@ -13,39 +13,89 @@ import (
...
+
+	for _, tc := range testCases {
+		name := fmt.Sprintf("version %v", tc.version)
+		t.Run(name, func(t *testing.T) {
+			got, err := appconsts.GlobalMinGasPrice(tc.version)
+			if tc.expErr {
+				require.Error(t, err)
+			} else {
+				require.NoError(t, err)
+				require.Equal(t, tc.expected, got)
```
Comment (staheri14) [inline pkg/appconsts/versioned_consts_test.go:69]: I think this line is meant to be under the `if` block.

Comment (ninabarbakadze) [reply pkg/appconsts/versioned_consts_test.go:69]: I thought it was sufficient enough to just check that the error was thrown. if you think that it's better if I take that line out of the `if`  block I'm more than happy to!

Comment (staheri14) [reply pkg/appconsts/versioned_consts_test.go:69]: I previously thought that `tc.expected` referred to the expected error. No modification is necessary, the current version is correct.

**Review by staheri14 — APPROVED** [review]
LGTM!

**Pull request merged**